### PR TITLE
Clarify steps to enable remote deployment from Visual Studio

### PR DESCRIPTION
**Pull request by dfields-msft**
Deploying to a HoloLens from Visual Studio will fail with obscure errors unless the **Device discovery** option is enabled on the HoloLens, in addition to "developer mode" (i.e. **Use developer features**).

### CODE REVIEW

**Comment by PRMerger10** [comment]
@dfields-msft : Thanks for your contribution! The author(s) have been notified to review your proposed change. @pbarnettms

File: mixed-reality-docs/mr-dev-docs/develop/platform-capabilities-and-apis/using-visual-studio.md [hunk 1]
```diff
@@ -32,8 +32,8 @@ Start by enabling **Developer Mode** on your device, so Visual Studio can connec
 3. Select the **Settings** tile to launch the app in your environment.
 4. Select the **Update** menu item.
 5. Select the **For developers** menu item.
-6. Enable **Developer Mode** to deploy apps from Visual Studio to your HoloLens.
-7. Optional: Scroll down and also enable **Device Portal**, which lets you connect to the [Windows Device Portal](using-the-windows-device-portal.md) on your HoloLens from a web browser.
+6. Enable **Use developer features** and **Device discovery** to deploy apps from Visual Studio to your HoloLens.
```
[the remark this answers is not in the document]
Comment (dfields-msft) [reply mixed-reality-docs/mr-dev-docs/develop/platform-capabilities-and-apis/using-visual-studio.md:35]: Ah, that's why I had trouble! I'm running 21H1. I suppose we'll need to clarify this to describe both configurations?

[the remark this answers is not in the document]
Comment (pbarnettms) [reply mixed-reality-docs/mr-dev-docs/develop/platform-capabilities-and-apis/using-visual-studio.md:35]: Yes.  Something like ""Enable 'Use developer features'.  If your device is running Windows Holographic version 21H1 or newer, also enable 'Device discovery' ..."

**Comment by PRMerger9** [comment]
@dfields-msft : Thanks for your contribution! The author(s) have been notified to review your proposed change. @pbarnettms

**Comment by PRMerger18** [comment]
@dfields-msft : Thanks for your contribution! The author(s) have been notified to review your proposed change. @pbarnettms

**Pull request merged**